.github/workflows/build.yml

```diff
@@ -91,9 +92,35 @@ jobs:
                     restore-keys: |
                         ${{ runner.os }}-php-${{ matrix.php }}-composer-
 
+            -
+                name: Prepare migrations for Sylius 1.7
+                if: matrix.sylius == '1.7.*'
+                run: |
+                    cp src/Migrations/* tests/Application/src/Migrations/
+                    cd tests/Application/src/Migrations
+                    sed -i -- 's/Sylius\\RefundPlugin\\Migrations/DoctrineMigrations/g' *
+                    cd ../../../../
+
+            -   name: Prepare migrations for Sylius 1.8
+                if: matrix.sylius == '1.8.*'
+                run: |
+                    composer require sylius-labs/doctrine-migrations-extra-bundle --no-interaction --no-update
+                    cp etc/travis/sylius18/bundles.php tests/Application/config/
```

Comment:
I wonder if it shouldn't be done the other way around, to have test application configured for 1.8 and prepared configs for 1.7 in `etc/` directory

Reply:
It makes sense 🖖 :D